clang/examples/DPCT/Driver/cuCtxGetLimit.cu

```diff
@@ -0,0 +1,5 @@
+void test(size_t *ps) {
+  // Start
+  cuCtxGetLimit(ps /*size_t **/, CU_LIMIT_PRINTF_FIFO_SIZE);
```

Comment:
```suggestion
  cuCtxGetLimit(ps /*size_t*/, CU_LIMIT_PRINTF_FIFO_SIZE);
```

Reply:
It is a pointer, currently, the `*` connected with the `*/`, may optimize later.